app/actions/remote/entry/common.ts

```diff
@@ -290,6 +292,10 @@ export async function deferredAppEntryActions(
 
     await fetchAllTeams(serverUrl);
     await updateAllUsersSince(serverUrl, since);
+
+    // Fetch groups for current user
+    fetchGroupsForMember(serverUrl, currentUserId);
```

Comment:
Out of scope:
Does it make sense to find a way to get this into GraphQL? What would be needed?

Reply:
That would be great - I can dig into that a bit later; but essentially it is the

- [Group Model](https://github.com/mattermost/mattermost-server/blob/master/model/group.go)
- [Group Member Model](https://github.com/mattermost/mattermost-server/blob/master/model/group_member.go)

And down the track, even being able to get the Channel-Group and Team-Group associations would be great.